configuration/pih/pih-config-hinche-ci.json

```diff
@@ -0,0 +1,9 @@
+{
+  "components": [
+    "homeVisitsOnClinicianDashboard",
+    "j9"
+  ],
```

Comment:
Unfortuntely we don't have support yet for merging at the component level... ie, the components defined here will override all that are defined in pih-config-hinche, so you'll need to include the full list of components defined in pih-config-hinche.json here, plus these two new ones...